src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -3655,9 +3691,13 @@ const static std::unordered_map<uint32_t, GfxOpcodeHandlerFunc> s2dexHandlers =
     { G_RDPHALF_2, gfx_stubbed_command_handler },
 };
 
-const static std::array<const std::unordered_map<uint32_t, GfxOpcodeHandlerFunc>*, UcodeHandlers::ucode_max>
+const static std::array<const std::unordered_map<int8_t, GfxOpcodeHandlerFunc>*, UcodeHandlers::ucode_max>
     ucode_handlers = {
+#ifdef F3DEX_GBI_2
```

Comment:
I don't really think this array should really exist. We should define this when we initialize Fast3D as passed in arguments.